recipe/meta.yaml

```diff
@@ -1,24 +1,25 @@
 {% set name = "jupyter_events" %}
 {% set version = "0.6.3" %}
+{% set sha256 = "9a6e9995f75d1b7146b436ea24d696ce3a35bfa8bfe45e0c33c334c79464d0b3d" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_events-{{ version }}.tar.gz
-  sha256: 9a6e9995f75d1b7146b436ea24d696ce3a35bfa8bfe45e0c33c334c79464d0b3
+  sha256: {{ sha256 }}
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
     - pip
     - python >=3.7
```

Comment:
If we build an arch-specific python package then there should be a plain `python` without pinning in `host` and `run`, see https://github.com/anaconda-distribution/perseverance-skills/blob/8a5a21c8a0167ec2dd20e8210f4b9a6968a9fad9/sections/02_Package_building/02_Recipes/Recipe_standards.md?plain=1#L83